train.py

```diff
@@ -774,15 +774,17 @@ def main():
         _logger.info(
             f'Scheduled epochs: {num_epochs}. LR stepped per {"epoch" if lr_scheduler.t_in_epochs else "update"}.')
 
-    # setup mlflow tracking
     if has_mlflow:
-        experiment_name = args.model
-        run_name = datetime.now().strftime("%Y%m%d-%H%M%S")
-        experiment = mlflow.get_experiment_by_name(experiment_name)
-        if experiment:
-            experiment_id = experiment.experiment_id
-        else:
-            experiment_id = mlflow.create_experiment(experiment_name)
+        experiment_id = None
+        run_name = None
+        if not os.environ.get("MLFLOW_RUN_ID"):
```

Comment:
You can assume 'MLFLOW_RUN_ID' is set. 
Then, I think experiment_id, run_name are not needed anymore?

Reply:
If it's for another purpose, please let me know :) !

Reply:
Agreed. `ModelRunner` of `model_runner.py` will do this instead.

Reply:
I agree that it wouldn't be necessary. This is mostly for maintaining the ability to run with the script `run_batch.sh` too. I did this because it wasn't too complicated. Passing `experiment_id=None` and `run_name=None` to `mlflow.start_run` has no effect anyway.

Reply:
Then, no problem :)